website/screens/components/chip/code/ChipCodePage.tsx

```diff
@@ -29,22 +29,36 @@ const sections = [
           <tr>
             <td>prefixIcon: node | string</td>
             <td></td>
-            <td>Element used as icon to be placed before the chip label.</td>
+            <td>
+              Element used as icon, placed before the chip label. Note that if
+              the passed value is an URL (string), the component's color styling
+              tokens will not be applied to the image.
```

Comment:
Not any SVG will get color styling applied, they need to inherit them from the chip tokens.

Reply:
I'm not saying that only using SVG would make it work, but I can explain it better.